### DISCUSSION TITLE
后端接口怎么使用docker部署，package.json用的线上的包没有使用本地的

### DISCUSSION THREAD

**Discussion by linqianppb** [nocodb/nocodb#8364] [Q&A] [answered]
### Please confirm if feature request does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

无

### Suggested Solution

无

**Comment by mertmit** [comment, marked as answer]
I had to use translation so please let me know if it is not correct:
>How to use docker to deploy the back-end interface. The online package used in package.json does not use the local one

I believe you are talking about nc-lib-gui, we bundle it together with our package each release as you've noticed.
In order to locally replicate that behavior you need to
- `pnpm run build:copy` inside nc-gui to move latest UI changes to nc-lib-gui package
- `pnpm install ../nc-lib-gui` inside nocodb to link local nc-lib-gui instead of released one

**Reply by linqianppb** [reply]
nocodb里面是有pnpm install ../nc-lib-gui过后，下一步使用pnpm run  哪一个命名可以打包成项目，然后制作成docker镜像运行
